schemas/announcement_schema.py

```diff
@@ -0,0 +1,43 @@
+# schemas/announcement_schema.py
+from pydantic import BaseModel, root_validator, model_validator, ConfigDict
+from typing import Optional
+from datetime import datetime
+
+
+class AnnouncementCreate(BaseModel):
+    id: Optional[int] = None
+    title: str
+    description: str
+    announcement_date: datetime
+    announcement_deadline: Optional[datetime] = None
+    image: Optional[str] = None
+
+    class Config:
```

Comment:
this is deprecated. Use <model_config = ConfigDict(from_attributes=True)>

Reply:
Done! Thanks